e2e/src/test/groovy/gov/hhs/cdc/trustedintermediary/e2e/DemographicsTest.groovy

```diff
@@ -0,0 +1,117 @@
+package gov.hhs.cdc.trustedintermediary.e2e
+
+import spock.lang.Specification
+
+class DemographicsTest extends Specification {
+
+    def newbornPatient = """{
+          "resourceType": "Patient",
+          "id": "infant-twin-1",
+          "text": {
+            "status": "generated",
+            "div": "<div xmlns=\\"http://www.w3.org/1999/xhtml\\"><p><b>Jaina Solo (OFFICIAL)</b> female, DoB: 2017-05-15 ( Medical record number: MRN7465737865)</p></div>"
+          },
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/patient-mothersMaidenName",
+              "valueString": "Organa"
+            }
+          ],
+          "identifier": [
+            {
+              "type": {
+                "coding": [
+                  {
+                    "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
+                    "code": "MR"
+                  }
+                ]
+              },
+              "system": "http://coruscanthealth.org/main-hospital/patient-identifier",
+              "value": "MRN7465737865"
+            },
+            {
+              "system": "http://new-republic.gov/galactic-citizen-identifier",
+              "value": "7465737865"
+            }
+          ],
+          "name": [
+            {
+              "use": "official",
+              "family": "Solo",
+              "given": [
+                "Jaina"
+              ]
+            }
+          ],
+          "gender": "female",
+          "birthDate": "2017-05-15",
+          "_birthDate": {
+            "extension": [
+              {
+                "url": "http://hl7.org/fhir/StructureDefinition/patient-birthTime",
+                "valueDateTime": "2017-05-15T17:11:00+01:00"
+              }
+            ]
+          },
+          "multipleBirthInteger": 1,
+          "contact": [
+            {
+              "relationship": [
+                {
+                  "coding": [
+                    {
+                      "system": "http://snomed.info/sct",
+                      "code": "72705000",
+                      "display": "Mother"
+                    },
+                    {
+                      "system": "http://terminology.hl7.org/CodeSystem/v2-0131",
+                      "code": "N"
+                    },
+                    {
+                      "system": "http://terminology.hl7.org/CodeSystem/v3-RoleCode",
+                      "code": "MTH"
+                    }
+                  ]
+                }
+              ],
+              "name": {
+                "use": "maiden",
+                "family": "Organa",
+                "given": [
+                  "Leia"
+                ]
+              },
+              "telecom": [
+                {
+                  "system": "phone",
+                  "value": "+31201234567",
+                  "use": "mobile"
+                }
+              ]
+            }
+          ]
+        }"""
```

Comment:
Wonderful work. FHIR is very robust, what are your thoughts on using files where we put the mockMessage for the e2e tests to use?

Reply:
I like this idea.  Let's do that in some follow-on PR.